parser/fsimporter.go

```diff
@@ -61,12 +62,15 @@ func (fs *FSImporter) Run(datastore Datastore) {
 		},
 	).Info("Finished collecting file details. Starting upload.")
 
-	indexedFiles = removeOldFilesFromIndex(indexedFiles, fs.res.MetaDB, fs.res.Log)
+	indexedFiles = removeOldFilesFromIndex(indexedFiles, fs.res.FileIndex, fs.res.Log)
```

Comment:
The importer needed some changes after replacing the metadb class with the RITADatabaseIndex and FileIndex classes